src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -12,11 +12,9 @@ namespace Microsoft.DotNet.GenAPI
 {
     internal static class SyntaxGeneratorExtensions
     {
-        /// <summary>
-        /// Creates a declaration matching an existing symbol.
-        ///     The reason of having this similar to `SyntaxGenerator.Declaration` extension method is that
-        ///     SyntaxGenerator does not generates attributes neither for types, neither for members.
-        /// </summary>
+        // Creates a declaration matching an existing symbol.
+        // The reason of having this similar to `SyntaxGenerator.Declaration` extension method is that
+        // SyntaxGenerator does not generates attributes neither for types, neither for members.
         public static SyntaxNode DeclarationExt(this SyntaxGenerator syntaxGenerator, ISymbol symbol, ISymbolFilter symbolFilter)
```

Comment:
nit: The `*Ext` suffix in this extension method name is redundant/unnecessary. If it's not too disruptive, consider removing it. I am not sure if the name `Declaration` would cause issues.

Reply:
I'm not entirely certain, but I believe my wanted to highlight that the method is indeed an extension method. Feel free to rename it.